test/PayID/fakes/fake-pay-id-client.ts

```diff
@@ -0,0 +1,29 @@
+import PayIDClientInterface from '../../../src/PayID/pay-id-client-interface'
+
+/**
+ * A response for a request to retrieve type T. Either an instance of T, or an error.
+ */
+// TODO(keefertaylor): Deduplicate this type across `FakeNetworkClient` and this file.
+// TODO(keefertaylor): Rename this to `Result` which is more inline with what it actually is.
+type Response<T> = T | Error
```

Comment:
I wish TypeScript had `Result<T>` natively.

Reply:
Merged in changes from #280  which obsoleted this TODO.